www/index.php

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />
+	<title>TopoMath</title>
+	<link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon"/>
+	<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+	<script type = "text/javascript" src = "version.js"></script>
+	<script type="text/javascript">
+		var version = "";
+
+		if(window.location.hostname == "topomath.asu.edu"){
+			version = getVersion();  // Get version from version.js
+		}else{
+			version = new Date();
+		}
+
+		dojoConfig = {
+			isDebug:true,
```

Comment:
This has always been true. This is so in Dragoon as well. Put that in backlog I will make the change later.